frontend/src/components/common/export-recipients/ExportRecipients.tsx

```diff
@@ -203,11 +201,9 @@ const ExportRecipients = ({
     <>
       {isButton ? (
         <div className={styles.actionButton}>
-          <InfoBlock className={styles.notice}>
```

Comment:
is this design change intentional?

BEFORE:
![image](https://user-images.githubusercontent.com/67887489/186145456-87ca5a17-ef1b-4341-a881-107366c5377a.png)

AFTER:
![image](https://user-images.githubusercontent.com/67887489/186145475-0885f851-04f5-4101-ad81-1cee9041c0fa.png)

Reply:
Oh it is intentional nvm
<img width="542" alt="image" src="https://user-images.githubusercontent.com/67887489/186147361-c763ec5d-ae15-4fb4-91ee-b829c6e8f391.png">